src/current/cockroachcloud/plan-your-cluster.md

```diff
@@ -8,163 +8,99 @@ cloud: true
-  * *Workloads are unpredictable* \- OLTP workloads are remarkably diverse in their resources utilization across industries and applications  
-  * *Architectural differences* \- Standard and Advanced are built on different underlying CockroachDB deployment architectures that have different performance characteristics
+### More details about capacity
 
-### Standard vCPUs are priced at 60% of Advanced vCPUs
+We recommend that you start with [Quickstart: Provision and adjust capacity](#quickstart-provision-and-adjust-capacity) to arrive at the best initial compute capacity for your workload and that you adjust the cluster's capacity over time as the workload evolves. In addition, consider the following:
 
-The pricing per-vCPUs does not depend in any way on the workload. Standard vCPUs are priced at 60% of the price of Advanced vCPUs, with some slight variations depending on the cloud provider and region. Since vCPUs are equivalent between Standard and Advanced on average, both plans require the same number of vCPUs on average, so total cluster cost for Standard is 60% of Advanced *on average*.
+- **Smaller workloads**: CockroachDB {{ site.data.products.standard }} is well suited for workloads that require 12 or fewer vCPUs. By contrast, 12 vCPUs (3 nodes x 4 vCPUs) is the minimum configuration for CockroachDB {{ site.data.products.advanced }}.
 
-But workloads may vary significantly from the average, so the estimation challenge is one of predicting the actual number of vCPUs a workload may require, regardless of whether you’re considering Standard or Advanced.
+  CockroachDB {{ site.data.products.standard }} is not recommended for:
+    - Analytical workloads that involve a large quantity of large, complex reads
+    - OLAP or hybrid OLTP/OLAP applications
 
-### Comparing total cluster vCPU capacity cost between Standard and Advanced
+- **Asymmetric multi-region workloads**: Often, multi-region deployments are asymmetric in terms of CPU utilization across regions. For example, in a three-region cluster, two of the three regions may require more capacity than the third, or vice versa. In CockroachDB {{ site.data.products.standard }}, a cluster's compute capacity is available in all regions.
 
-Given that Standard vCPUs are priced at 60% of Advanced vCPUs, estimating the cost of a workload on Standard as compared to Advanced means estimating the difference in number of vCPUs required between the two plans. For some workloads, Standard requires higher vCPU capacity, for others Advanced, depending on the properties of the workload and how the different underlying CockroachDB deployment architectures provide resources to that workload.
+  For workloads that require symmetric capacity per region or where a given region must maintain a given compute capacity, CockroachDB {{ site.data.products.advanced }} may be a better option.
```

Comment:
1. symmetric -> asymmetric?
2. This isn't showing up as a second paragraph of the same bullet. I suspect the blank line after is to blame; that the whole list should have no blank spaces except for the space *above* this line. But I'm not sure.